index.html

```diff
@@ -14,15 +14,18 @@
         <nav class="nav-list">
             
             <a class="logo" href="#">Leslie Gudo</a>
-            <ul class="navbar-list">
-
-            <li class="menu-icon">
-                <img class="menu" src="static/images/Union.png" alt=" navigation" />
-            </li>
-            <li class="desktop-nav-list">Portfolio</li>
-            <li class="desktop-nav-list">About</li>
-            <li class="desktop-nav-list">Contact</li>
+            <div id="modal" class="modal hidden">
+              <span id="modalClose" class="modalClose">&times;</span>
+            <ul id="navLinks" class="navbar-list">
+            <li class="desktop-nav-list" id="portfolio-1"><a href="#">Portfolio</a></li>
+            <li class="desktop-nav-list" id="about-1"><a href="#about">About</a></li>
+            <li class="desktop-nav-list" id="contact-1"><a href="#contact">Contact</a></li>
```

Comment:
- [ ] As mentioned by the previous reviewer and for the project requirements, kindly ensure that when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. Right now, when clicking the `Portfolio` option, it displays the the `Hero` section instead of the `Works` section. Make sure you move your `#portfolio-1` id to the correct section to be displayed when clicking on it.